web-client/slycat/web/client/dac_run_chart.py

```diff
@@ -357,10 +357,11 @@ def read_metadata_table (metadata, arguments, log):
     meta_column_names = ['Part', 'Lot', 'Batch', 'Serial Number', 'Source']
     for tdms_type in metadata[0]["tdms_types"]:
         meta_column_names += ['Module ID [' + tdms_type + ']']
-    for tdms_type in metadata[0]["tdms_types"]:
         meta_column_names += ['Module SN [' + tdms_type + ']']
-    for tdms_type in metadata[0]["tdms_types"]:
-        meta_column_names += ['Module ID/SN [' + tdms_type + ']']
+
+    # for tdms_type in metadata[0]["tdms_types"]:
+    #     meta_column_names += ['Module ID/SN [' + tdms_type + ']']
+

```

Comment:
we could probably just delete this right?

Reply:
Yes, that would be fine.

Reply:
I deleted it